shopelectro/views/catalog.py

```diff
@@ -33,6 +32,18 @@ def get_paginated_page_or_404(objects, per_page, page_number):
         raise http.Http404('Page does not exist')
 
 
+class CategorySortingOption:
```

Comment:
`SortingOption` may be sufficient, because we use it only in `catalog.py`

PS: by the way it seems it is about a catalog sorting and not about a category